containers/daap-python-base/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM public.ecr.aws/lambda/python:3.11
+
+ARG BASE_VERSION
+
+ENV BASE_VERSION="${VERSION}"
```

```suggestion
ARG VERSION

ENV BASE_VERSION="${VERSION}"
```

The argument passed in at build time is `VERSION` https://github.com/ministryofjustice/data-platform/blob/main/.github/workflows/reusable-workflow-containers.yml#L240